config/routes.rb

```diff
@@ -29,7 +29,7 @@
 
   scope "(:locale)", :locale => /#{Locales.available.keys.join('|')}/ do
 
-    devise_for :users, :controllers => {:registrations => 'users/registrations', omniauth_callbacks: "omniauth_callbacks" } do
+    devise_for :users, :skip => [ ( :registrations if Guisso.enabled? ) ], :controllers => {omniauth_callbacks: "omniauth_callbacks" } do
```

Comment:
Is this change equivalent? I mean, if Guisso is disabled does it default the registrations controller to `user/registrations`?

Reply:
Great catch! It was a mistake, yes.
I've just fixed it 👍